src/lib/utils/time.ts

```diff
@@ -52,11 +52,13 @@ export const categorizeDate = (date: Date): string | number => {
     millisecondsSince(date) <= lastThirtyDays
   ) {
     return 'lastMonth'
+
```

Comment:
Try to avoid putting empty lines at the start or end of a block (right before or after `{` or `}`) it makes the code harder to read in my opinion 🙂